### DISCUSSION TITLE
Cloud plugin for not supported Steam cloud games

### DISCUSSION THREAD

**Discussion by jaimeamate** [SteamDeckHomebrew/decky-loader#619] [Ideas]
Hi, i've been recently start playing dark souls III in my Pc and when i want resume my game in Steam deck i realized this game is not saving my game in Steam cloud so the only way i found to continue in Steam deck and both is exporting the saves file/s and importing to the same route in Steam deck or any device. I would be very nive have a plugin that allows you to save this files in your Google or One cloud or another cloud making this problem tinnier. 
Thanks for reading.

**Comment by TrainDoctor** [comment]
Such plugins already exist and discussions/issues are not to be used for plugin requests/suggestions
